### PR TITLE
Issue #14: Remove 'code' tag from PostReadingTime component

### LINKED ISSUE #14: Remove 'code' tag from PostReadingTime component
PostReadingTime component is responsible for showing the time it takes to read this post.
This component should ignore "code" blocks in the post and calculate only words to read.

### PR DESCRIPTION
**Pull request by Siddharth1010**
Fixes #14 Removed code tag from blocks

### CODE REVIEW

File: components/PostReadingTime.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { calculateReadingTime } from 'utils/helpers'
 
 export default function PostReadingTime({ blocks }) {
   const words = blocks.reduce((words, block, i) => {
-    block.html.trim().split(' ').forEach(word => word && words.push(word))
+    if(block.tag!=="code") block.html.trim().split(' ').forEach(word => word && words.push(word))
```
Comment (rsneh) [inline components/PostReadingTime.js]: Please keep the style guide.
Space between the if and "(", and also between expressions and operators

**Review by rsneh — CHANGES_REQUESTED** [review]
Please check my comment in the diff

**Comment by Siddharth1010** [comment]
Is it alright now?

**Review by rsneh — APPROVED** [review]

**Comment by rsneh** [comment]
All merged.
Thanks @Siddharth1010

**Comment by Siddharth1010** [comment]
Thank you

**Pull request merged**